rflx/model.py

```diff
@@ -1239,6 +1247,39 @@ def __prove_field_positions(self) -> None:
                     )
                     return
 
+                if f in self.__types and isinstance(self.__types[f], Opaque):
+                    start_aligned = Not(
+                        Equal(Mod(self.__target_first(last), Number(8)), Number(1), last.location)
+                    )
+                    proof = start_aligned.check([*facts, *self.__type_constraints(start_aligned)])
+                    if proof.result != ProofResult.unsat:
+                        path_message = " -> ".join([p.target.name for p in path])
+                        self.error.append(
+                            f'opaque field "{f.name}" not aligned to 8 bit boundary'
+                            f" ({path_message})",
+                            Subsystem.MODEL,
+                            Severity.ERROR,
+                            self.location,
```

Comment:
It would be nice if the location of the field would be used instead of just the location of the message type (same below).

Reply:
Fixed in 037393f.